src/planscape/planning/urls.py

```diff
@@ -79,4 +80,25 @@
         get_shared_link,
         name="get_shared_link",
     ),
+    # Planning Area Notes C/R/D
+    path(
+        "planning_area/<int:planningarea_pk>/note",
+        PlanningAreaNotes.as_view(),
+        name="create_planningareanote",
```

Comment:
I thought it might be helpful to migrate to a more RESTful url path here, but maybe it's just a confusing waste of time, since we're not even following this pattern for `planning_area` at the moment.